command/run-document.go

```diff
@@ -0,0 +1,76 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"github.com/itsdalmo/ssm-sh/manager"
+	"github.com/pkg/errors"
+	"os"
+	"time"
+)
+
+type RunDocumentCommand struct {
+	Name       string   `short:"n" long:"name" description:"Name of document in ssm."`
+	Timeout    int      `short:"i" long:"timeout" description:"Seconds to wait for command result before timing out." default:"30"`
+	Parameters []string `short:"p" long:"parameter" description:"Zero or more parameters for the document (name=value)"`
+	TargetOpts TargetOptions
+}
+
+func (command *RunDocumentCommand) Execute(args []string) error {
+	sess, err := newSession()
+	if err != nil {
+		return errors.Wrap(err, "failed to create new aws session")
+	}
+
+	if command.Name == "" {
+		return errors.New("No document name set to trigger")
+	}
+
+	m := manager.NewManager(sess, Command.AwsOpts.Region)
```

Comment:
This part is very similar to the regular `RunCommand` (cli), so I'm thinking that it should perhaps become a helper function which is used both in `RunCommand` and here?